OutOfSchool/OutOfSchool.BusinessLogic/Models/Tag/TagDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.BusinessLogic.Models.Tag;
+public class TagDto
+{
+    public long Id { get; set; }
+
+    [MaxLength(60)]
```

Comment:
Maybe we can move these magic lengths to OutOfSchool.Common.Constants?

Reply:
What is OutOfSchool.Common.Constants?
![image](https://github.com/user-attachments/assets/04574fda-5618-4609-a0e3-dacc9f51f79b)

Reply:
![image](https://github.com/user-attachments/assets/f00dc1c8-571e-4cf0-ba49-6db2d4649afc)